src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -218,7 +263,7 @@ public static partial class AdvApi32
         /// </returns>
         [DllImport(api_ms_win_service_management_l2_1_0, SetLastError = true, CharSet = CharSet.Unicode)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        public static unsafe extern bool ChangeServiceConfig2(SafeServiceHandle hService, ServiceInfoLevel dwInfoLevel, void* lpInfo);
+        public static extern bool ChangeServiceConfig2(SafeServiceHandle hService, ServiceInfoLevel dwInfoLevel, IntPtr lpInfo);
```

Comment:
I am really sorry for this, but as the last AppVeyor build from this PR states all the messages are the motivator to the changes on the  commit bf30bd97dfedb1ff2a9c75e5872b629c3a527e2e . I did all the changes on a single commit so that we could reverse it later when it could be discussed and solved. I am reverting now and will leave so it the build breaks. Unfortunately I didn't understand why it did happen because I didn't even changed these files when it started to happen.